work-list-page.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="./css/work-page.css">
+    <title>Recent Works</title>
+</head>
```

Comment:
- [ ] I can't see the importing of the "inter" font family.  Kindly please import it and use it where it's required.
- [ ] Also, in your imports you need to include the font-weights that you are going to use, otherwise, your font weights on the CSS will get ignored by the browser (since they don't exist in the import).

 1 - First, go [here](https://fonts.google.com/specimen/Inter) for the inter.  
 2 - Then, select all the font-weights you are going to need.
 3 - Then, copy the resulting `<link>`'s to the `<head>` section of your project.
  
Look: 

![image](https://user-images.githubusercontent.com/57421823/149504208-434d626f-d24c-461c-8358-dc8c3a7fbec8.png)

![image](https://user-images.githubusercontent.com/57421823/149504663-1380e30b-48a5-4f56-b46d-36e47e1af078.png)

 4 - Delete your existing `<link>` because now it's a duplicate:

![image](https://user-images.githubusercontent.com/57421823/149504886-a74689d3-442b-4421-9f65-3552b0f69eb0.png)


 5- Do the same steps for the other font "Poppins".

That way, your fotnt-weights will actually show, and also the "inter" font family!! 💪 🥇